test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/CreateTests.cs

```diff
@@ -263,6 +263,19 @@ public async Task GivenAResource_WhenPostingToHttpWithMaliciousNarrative_TheServ
             }
         }
 
+        [Fact]
+        [HttpIntegrationFixtureArgumentSets(DataStore.SqlServer, Format.All)]
```

Comment:
Should we have a test to show what happens when a Reindex is run on a database with a value for the new search parameter that is too long?